web/src/styles/scss/section.scss

```diff
@@ -28,8 +28,13 @@ body {
     }
   }
 
-  margin-top: $spacer-2;
+  margin-top: $spacer-6;
```

Comment:
I added a little extra padding to the regular `Section` component here, because it felt like it needed more breathing space once the subsection was uncollapsed.